content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -11,8 +11,25 @@ further_reading:
     - link: 'https://www.datadoghq.com/blog/monitor-otel-with-w3c-trace-context/'
       tag: 'Blog'
       text: 'Monitor OpenTelemetry-instrumented apps with support for W3C Trace Context'
+    - link: '/opentelemetry/guide/otel_api_tracing_interoperability'
+      tag: 'Documentation'
+      text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
+W3C trace propagation and 128 bit headers is available for all supported languages. The default trace propagation style for each language is:
+
+| Language | Default inject and extract setting |
+| -------- | ---------------------------------- |
+| Java     | `datadog,tracecontext`             |
+| Python   | `tracecontext,datadog`             |
+| Ruby     | `datadog,tracecontext`             |
```

Comment:
It looks like Ruby by default will _extract_ `Datadog,b3multi,b3,tracecontext` and then _inject_ `Datadog,tracecontext`. Since the extract operation is the one that affects propagation the most, I suggest this small update:

```suggestion
| Ruby     | `Datadog,b3multi,b3,tracecontext`             |
```

Reply:
@marcotc can you also confirm this change?

Reply:
This is correct for Ruby, thank you @zacharycmontoya!